std_msgs/String type
geometry_msgs/Vector3 location